server/templates/staff/jobs/github_search.html

```diff
@@ -0,0 +1,47 @@
+{% extends "staff/base.html" %}
+{% import "staff/_formhelpers.html" as forms %}
+
+{% block title %} Github Search | Ok {% endblock %}
+
+{% block main %}
+<section class="content-header">
+    <h1>
+        {{ current_course.display_name_with_semester }} Jobs
+        <small>{{ current_course.offering }}</small>
+    </h1>
+    <ol class="breadcrumb">
+        <li><a href="{{ url_for(".index") }}"><i class="fa fa-dashboard"></i> Home</a></li>
+        <li><a href="{{ url_for(".course", cid=current_course.id) }}">
+            <i class="fa fa-university"></i> {{ current_course.offering }}
+        </a></li>
+        <li><a href="{{ url_for('.course_assignments', cid=current_course.id) }}">
+          <i class="fa fa-list"></i> Assignments</a>
+        </li>
+        <li> <a href="{{ url_for('.assignment', cid=current_course.id, aid=assignment.id) }}"><i class="fa fa-book"></i> {{ assignment.display_name }} </a></li>
+        <li><a href="{{ url_for(".course_jobs", cid=current_course.id) }}">
+            <i class="fa fa-list"></i>Jobs
+        </a></li>
+        <li class="active"><a href="{{ url_for('.start_github_search', cid=current_course.id, aid=assignment.id) }}">
+            <i class="fa fa-github"></i>Github Search</a>
+        </li>
+    </ol>
+</section>
+<section class="content">
+  {% include 'alerts.html' %}
+  <div class="row">
+    <div class="col-md-12">
+      <div class="box">
+        <div class="box-body">
+          {% call forms.render_form(form, action_text='Search Github') %}
+            {{ forms.render_field(form.access_token) }}
+            {{ forms.render_field(form.template_name, placeholder="hog.py") }}
+            {{ forms.render_field(form.weeks_past, default=12) }}
+            {{ forms.render_field(form.keyword, default="def ") }}
+            {{ forms.render_field(form.language) }}
+          {% endcall %}
```

Comment:
It seems like all of these jobs are just the same thing - render the form. Is there an easy way to just render the form with one call? If so we could generalize this template.